AEPServices/Sources/ApplicationSystemInfoService.swift

```diff
@@ -26,6 +26,9 @@ class ApplicationSystemInfoService: SystemInfoService {
     private let DEFAULT_LOCALE = "en-US"
 
     private let bundle: Bundle
+
+    private let queue = DispatchQueue(label: "com.adobe.applicationSystemInfoService.queue")
```

Comment:
unrelated to this change, but can we also have the ServiceProvider and EventHistoryDatabase queue labels prefixed with `com.adobe.`

Reply:
I will make the names consistent it in a different PR.